packages/cms/express-mini-apps/two-factor-auth/route.ts

```diff
@@ -56,6 +57,15 @@ export function twoFactorAuthRoute(
     // generate secret and get QR code
     app.get('/api/2fa/setup', async (req, res) => {
       const context = await commonContext.withRequest(req, res)
+
+      if (
+        !req.cookies['keystonejs-2fa'] ||
+        !verify2FAJWT(req.cookies['keystonejs-2fa'], context.session.itemId)
+      ) {
+        res.status(403).send({ status: 'error', message: 'invalid 2fa' })
```

Comment:
suggestion:
403 is a client side error. In jSend format, it should be `status: 'fail'` instead.

Reply:
done